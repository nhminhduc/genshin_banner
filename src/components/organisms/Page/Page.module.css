.pageBackground::before {
  content: "";
  position: absolute;
  top: 0;
  left: 0;
  width: 100vw;
  height: 100vh;
  opacity: 0.4;
  z-index: -1;
  background-image: url("/bg.webp");
  background-repeat: no-repeat;
  background-size: cover;
}
