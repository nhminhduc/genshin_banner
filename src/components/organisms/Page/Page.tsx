import VersionFromLastBanner from "../../../utils/fromLastBanner";

const Page = () => {
  const abc = VersionFromLastBanner({
    "last_date": "2022-10-14",
    "version": "3.1.2"
  });
  return (<div>{abc}</div>)
};

export default Page;
