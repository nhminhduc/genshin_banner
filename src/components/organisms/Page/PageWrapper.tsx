import { createContext } from "react";
import { useCharacterData } from "hooks/useCharacterData";
import Page from "./Page";

export const CharacterDataContext = createContext<CharacterData[]>([{} as CharacterData]);
const PageWrapper = () => {
  const characterData = useCharacterData();
  return (
    <CharacterDataContext.Provider value={characterData}>
      <Page />
    </CharacterDataContext.Provider>
  )
};

export default PageWrapper;
