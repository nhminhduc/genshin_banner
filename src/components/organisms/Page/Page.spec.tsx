import { describe, expect, test } from "vitest";
import { render } from "@testing-library/react";
import Page from "./Page";

describe("Page test", () => {
  test("Page load", () => {
    const { asFragment } = render(<Page />);
    expect(asFragment).toMatchSnapshot();
  });
});
