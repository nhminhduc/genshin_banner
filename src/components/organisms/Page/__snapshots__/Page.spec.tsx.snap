// Vitest Snapshot v1

exports[`Page test > Page load 1`] = `[Function]`;
